src/App.tsx

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [pressedKey, setPressedKey] = useState('');
+
+  useEffect(() => {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      setPressedKey(event.key);
+    });
+
+    return () => {
+      document.removeEventListener('keyup', (event: KeyboardEvent) => {
+        setPressedKey(event.key);
+      });
+    };
```

Comment:
you need to create your callback as a separate function in order to be able to remove a listener